test/controllers/answers_controller_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
```

Comment:
If all these tests have passed, I don't need to write mine again. But I forgot to mention it the last time, I changed our test framework to rspec. Right from the beginning when I pulled. @andela-ooranagwa  and @andela-aadepoju are aware though.